lua/ulx/modules/sh/cfc_owoify.lua

```diff
@@ -0,0 +1,78 @@
+local CATEGORY_NAME = "Chat"
+local PHRASES_TO_APPEND = {
+    " ^w^",
+    " :3",
+    " :3c",
+    " ;;w;;",
+    " x3",
+    " owo", 
+    " nya~",
+    " uwu", 
+    "~"
+}
+local PHRASES_TO_REPLACE = {
+    { "oo", "ew" },
+    { "O", "owo" },
+    { "U", "uwu" },
+    { "lol", "uwu" },
+    { "lmao", "uwu" },
+    { "lmfao", "uwu" },
+    { "l", "w" },
+    { "na", "nya" },
+    { "ne", "nye" },
+    { "ni", "nyi" },
+    { "no", "nyo" },
+    { "nu", "nyu" },
+    { "ove", "uv" },
+    { "r", "w" },
+    { "tion", "shun" },
+    { "wha", "wa" },
+    { "whe", "we" },
+    { "whi", "wi" },
+    { "you", "nyu" }
+}
+
+local function owoifyMessage( message )
+    local owoifiedMessage = message
+    
+    for _, item in pairs( PHRASES_TO_REPLACE ) do
+        local old = item[1]        
+        local new = item[2]
+        owoifiedMessage = string.Replace( owoifiedMessage, old, new )
+    end
+    owoifiedMessage = owoifiedMessage .. PHRASES_TO_APPEND[math.random( 1, #PHRASES_TO_APPEND )]
+
+    return owoifiedMessage
+end
+
+local function onPlayerSay( ply, message )
+    if not ply.isOwoified then return end
+
+    return owoifyMessage( message )
+end
+
+local function owoifyCommand( caller, targets, shouldUnowoify )
+    for _, ply in pairs( targets ) do
+        
+        if shouldUnowoify then
+            ply.isOwoified = nil
+        else
+            ply.isOwoified = true
+        end
```

Comment:
all this could be written as 1 lone
`ply.isOwoified = not shouldUnowoify or nil`

Reply:
Sure.

Reply:
would this apply to the fancylog stuff as well?